applications/nublado/templates/controller-ingress-files.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: "nublado-controller-files"
+  labels:
+    {{- include "nublado.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    all:
+      - "exec:notebook"
```

Comment:
Now you don't have a delegated token at all. Don't you need the user's UID, GID, and group information to spawn their fileserver?